src/caduceus/outboundSender_test.go

```diff
@@ -0,0 +1,263 @@
+package main
```

Comment:
Another thing I was going to mention was that you're not really using any interfaces in `outboundSender.go` but I guess you don't need to worry about that unless you needed to do some dependency injections for your unit tests.